.eslintignore

```diff
@@ -1,5 +1,6 @@
 node_modules/*
 modules/*/js/*.js
+modules/*/js/*/*.js
```

Comment:
Maybe lines 2 and 3 could just be `modules/*/js/*`?

Reply:
removed the `.js` at the end of the line 2 and 3.